test/Swan.Test/ConnectionExtensionsTest.cs

```diff
@@ -119,16 +119,118 @@ public void CreateProjectTableAndInsertOneRow()
+        conn.TableBuilder<Project>("ProjectsTableOne").ExecuteDdlCommand();
+        conn.TableBuilder<Project>("ProjectsTableTwo").ExecuteDdlCommand();
+        conn.TableBuilder<Project>("ProjectsTableThree").ExecuteDdlCommand();
+
+        var tableNames = conn.GetTableNames();
+       
+        Assert.IsTrue(tableNames.Any());
+        Assert.IsTrue(tableNames.Count == 3);
+    }
+
+    [Test]
+    public async Task CreateTablesAndGetTableNamesAsync()
+    {
+        var conn = new SqliteConnection("Data Source=:memory:");
+        await conn.TableBuilder<Project>("ProjectsTableOne").ExecuteDdlCommandAsync();
+        await conn.TableBuilder<Project>("ProjectsTableTwo").ExecuteDdlCommandAsync();
+        await conn.TableBuilder<Project>("ProjectsTableThree").ExecuteDdlCommandAsync();
+
+        var tableNames = conn.GetTableNamesAsync();
+
+        Assert.IsTrue(tableNames.Result.Any());
+        Assert.IsTrue(tableNames.Result.Count == 3);
+    }
+
+    [Test]
+    public async Task CreateProjectsTableAndInsertsQueryAsync()
+    {
+        var conn = new SqliteConnection("Data Source=:memory:");
+        await conn.TableBuilder<Project>("Projects").ExecuteDdlCommandAsync();
+
+        var execution = await conn.
+            ExecuteNonQueryAsync($"Insert into Projects (CompanyId, EndDate, IsActive, Name, ProjectScope, ProjectType, StartDate)" +
+            " values " +
+            $" (1,'{DateTime.Now}','{true}','Project ONE','My Scope', '{ProjectTypes.Exciting}','{DateTime.Now.AddMonths(-1)}');");
+
+        Assert.IsTrue(execution == 1);
+    }
+
+    [Test]
+    public void CreateProjectsTableAndInsertsQuery()
+    {
+        var conn = new SqliteConnection("Data Source=:memory:");
+        conn.TableBuilder<Project>("Projects").ExecuteDdlCommand();
+
+        var execution = conn.
+            ExecuteNonQuery($"Insert into Projects (CompanyId, EndDate, IsActive, Name, ProjectScope, ProjectType, StartDate)" +
+            " values " +
+            $" (1,'{DateTime.Now}','{true}','Project ONE','My Scope', '{ProjectTypes.Exciting}','{DateTime.Now.AddMonths(-1)}');");
+
+        Assert.IsTrue(execution == 1);
+    }
+
+    [Test]
+    public async Task CreateProjectsTableAndInsertOneRowAndExecuteScalarAsync()
+    {
+        var conn = new SqliteConnection("Data Source=:memory:");
+        await conn.TableBuilder<Project>("Projects").ExecuteDdlCommandAsync();
+
+        var table = conn.TableAsync<Project>("Projects");
+        var project = await table.Result.InsertOneAsync(new()
+        {
+            CompanyId = 1,
+            EndDate = DateTime.Now,
+            IsActive = true,
+            Name = "Project ONE",
+            ProjectScope = "My Scope",
+            ProjectType = ProjectTypes.Exciting,
+            StartDate = DateTime.Now.AddMonths(-1)
+        });
+
+        var scalar = await conn.ExecuteScalarAsync("select Name, ProjectScope, ProjectType from Projects");
```

Comment:
This query is kinda wrong. When you fetch a scalar query, it should return only one value. And you are selecting multiple columns.